frontend/common/src/components/UserProfile/ProfileSections/DiversityEquityInclusionSection.tsx

```diff
@@ -68,35 +100,153 @@ const DiversityEquityInclusionSection: React.FunctionComponent<{
                   "Label preceding what groups the user identifies as part of, followed by a colon",
               })}{" "}
             </p>{" "}
-            <ul
-              data-h2-font-weight="base(700)"
-              data-h2-padding="base(0, 0, 0, x1)"
+            {(isIndigenous ||
+              (indigenousCommunities && indigenousCommunities.length > 0)) && (
+              <div
+                data-h2-padding="base(x1, 0, 0, 0)"
+                data-h2-display="p-tablet(flex)"
+                data-h2-align-items="base:(center)"
+                data-h2-justify-content="base:(space-between)"
+              >
+                <ul data-h2-padding="base(0, 0, 0, x1)">
+                  <li>
+                    {intl.formatMessage(
+                      getEmploymentEquityStatement("indigenous"),
+                    )}
+                  </li>
+                  <ul data-h2-padding="base(0, 0, 0, x1)">
+                    <li>
+                      {intl.formatMessage({
+                        defaultMessage: "My Community(ies):",
+                        id: "Mw4ShP",
+                        description:
+                          "Label preceding text listing the communities the user is a member of, followed by a colon",
+                      })}{" "}
+                      {nonLegacyIndigenousCommunitySelected
+                        ? indigenousCommunities.map((community, index) => {
+                            if (
+                              !community ||
+                              community ===
+                                IndigenousCommunity.LegacyIsIndigenous
+                            ) {
+                              return "";
+                            }
+
+                            const text = intl.formatMessage(
+                              getIndigenousCommunity(community),
```

Comment:
I think the communities need to be bolded.  From the design:
![image](https://user-images.githubusercontent.com/8978655/216629110-4c16b61f-3602-4a26-a8b0-be71b5e68a2e.png)
I think the affirmation statement should be bolded like the other EE statements, though the design doesn't have that.

Reply:
Another design issue for @BrookeCameron, the OTHER copy reads very strangely beside the other options.
![image](https://user-images.githubusercontent.com/8978655/216630524-01e54a06-f5ec-44e1-b52d-00a9bce60775.png)

Reply:
Try changing to bulleted list of communities
```
- "I affirm..."
  - Non-status First Nations
  - I am Indigenous and I don't see my community here
```

Reply:
Changed with [4168dde](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5494/commits/4168dde7281ffe73af7403e1fbdf996313678511).

Reply:
:heart: